flang/test/Driver/time-report.f90

```diff
@@ -0,0 +1,22 @@
+! Check that -ftime-report flag is passed as-is to fc1. The value of the flag
+! is only checked there. This behavior intentionally mirrors that of clang.
+!
+! RUN: %flang -### -c -ftime-report %s 2>&1 | FileCheck %s --check-prefix=CHECK-DRIVER
```

Comment:
do you need -mllvm -sort-timers=0 to prevent reordering of timers in the TimerGroup?